HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -602,20 +602,35 @@ def self.apply_attics(hpxml)
   end
 
   def self.apply_foundations(hpxml)
-    return unless hpxml.has_location(HPXML::LocationCrawlspaceVented)
+    if hpxml.has_location(HPXML::LocationCrawlspaceVented)
+      vented_crawls = hpxml.foundations.select { |f| f.foundation_type == HPXML::FoundationTypeCrawlspaceVented }
+      if vented_crawls.empty?
+        hpxml.foundations.add(id: 'VentedCrawlspace',
+                              foundation_type: HPXML::FoundationTypeCrawlspaceVented)
+        vented_crawls << hpxml.foundations[-1]
+      end
+      vented_crawls.each do |vented_crawl|
+        next unless vented_crawl.vented_crawlspace_sla.nil?
 
-    vented_crawls = hpxml.foundations.select { |f| f.foundation_type == HPXML::FoundationTypeCrawlspaceVented }
-    if vented_crawls.empty?
-      hpxml.foundations.add(id: 'VentedCrawlspace',
-                            foundation_type: HPXML::FoundationTypeCrawlspaceVented)
-      vented_crawls << hpxml.foundations[-1]
+        vented_crawl.vented_crawlspace_sla = Airflow.get_default_vented_crawl_sla()
+        vented_crawl.vented_crawlspace_sla_isdefaulted = true
+        break # EPvalidator.xml only allows a single ventilation rate
+      end
     end
-    vented_crawls.each do |vented_crawl|
-      next unless vented_crawl.vented_crawlspace_sla.nil?
 
-      vented_crawl.vented_crawlspace_sla = Airflow.get_default_vented_crawl_sla()
-      vented_crawl.vented_crawlspace_sla_isdefaulted = true
-      break # EPvalidator.xml only allows a single ventilation rate
+    if hpxml.has_location(HPXML::LocationManufacturedHomeUnderBelly)
+      belly_and_wing_foundations = hpxml.foundations.select { |f| f.foundation_type == HPXML::FoundationTypeBellyAndWing }
+      if belly_and_wing_foundations.empty?
+        hpxml.foundations.add(id: 'BellyAndWing',
+                              foundation_type: HPXML::FoundationTypeBellyAndWing)
+        belly_and_wing_foundations << hpxml.foundations[-1]
+      end
+      belly_and_wing_foundations.each do |foundation|
+        if foundation.belly_wing_skirt_present.nil?
+          foundation.belly_wing_skirt_present_isdefaulted = true
+          foundation.belly_wing_skirt_present = true
```

Comment:
This is an edge case, but need to add a `break` here like is done in the above example for vented crawlspace. If a HPXML file is submitted with two Foundation elements, neither of which have SkirtPresent specified, they would both default to true in the `in.xml`. But then if you tried to run that file, you'd get an error because [we only allow a single SkirtPresent](https://github.com/NREL/OpenStudio-HPXML/blob/mobile_home_belly_ducts/HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml#L544).

(Yeah, it's stupid that we can't allow a HPXML file w/ SkirtPresent=true showing up twice. If you can figure out a better way to handle this, I'm all ears. We're obviously trying to protect against the situation where you have SkirtPresent=true and SkirtPresent=false in the same file.)